### PR TITLE
Promote CancellationException to stable #KT-41837

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

File: backend.native/tests/build.gradle [hunk 1]
```diff
@@ -4509,7 +4509,6 @@ if (isAppleTarget(project)) {
             artifact = frameworkArtifactName
             library = libraryName
             isStatic = true
-            opts = ['-Xstatic-framework', "-Xopt-in=kotlin.ExperimentalStdlibApi"]
```
Comment (qwwdfsad) [inline backend.native/tests/build.gradle:4512]: @SvyatoslavScherbina PTAL

**Review by qwwdfsad — APPROVED** [review]
LGTM modulo pending question about `WasExperimental`

File: backend.native/tests/build.gradle (hunk 1 above)
Comment (SvyatoslavScherbina) [reply backend.native/tests/build.gradle:4512]: Should be ok if the test works `¯\_(ツ)_/¯`

**Review by SvyatoslavScherbina — APPROVED** [review]
Thanks!

**Review by SvyatoslavScherbina — APPROVED** [review]

**Pull request merged**